pkg/operator/ceph/file/health.go

```diff
@@ -83,31 +84,58 @@ func (c *mirrorChecker) checkMirroring(context context.Context) {
 			logger.Debugf("checking filesystem mirroring status %q", c.namespacedName.Name)
 			err := c.checkMirroringHealth()
 			if err != nil {
-				c.updateStatusMirroring(nil, nil, err.Error())
+				c.updateStatusMirroring(nil, nil, nil, err.Error())
 				logger.Debugf("failed to check filesystem %q mirroring status. %v", c.namespacedName.Name, err)
 			}
 		}
 	}
 }
 
 func (c *mirrorChecker) checkMirroringHealth() error {
```

Comment:
Is this goroutine for any update to the filesystem status? Maybe this needs to be renamed?
```suggestion
func (c *mirrorChecker) checkFilesystemHealth() error {
```